okhttp/src/main/java/okhttp3/internal/InternalKt.kt

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
weird filename. guess there's already `Internal.java`?

Reply:
Yeah. This will eventually get folded with that.